components/vf-show-more/vf-show-more.njk

```diff
@@ -0,0 +1,48 @@
+{# Make sure any variables are listed inside the following if statement #}
+{% if context %}
+  {# {% set vf-show-more = context.vf-show-more %} #}
+{% endif %}
+
+{# You can do Nunjucks logic in your templates that references the .yml settings
+   What's Nunjucks: https://mozilla.github.io/nunjucks/templating.html #}
+{#
+{% if item.image == "blank" %}
+  <span><!-- no image --></span>
+  {% else %}
+  {{ item.image }}
+{% endif %}
+#}
+
+<section data-vf-js-show-more data-vf-js-show-more-pager-size="3" class="vf-show-more">
+  {% render '@vf-section-header--default', {
+    "section_title": "Show more demo"
+  } %}
+
+  <div class="vf-grid vf-grid__col-3">
+    {% set items = [
+      { title: "foo", id: 1 },
+      { title: "foo", id: 2 },
+      { title: "foo", id: 3 },
+      { title: "foo", id: 4 },
+      { title: "foo", id: 5 },
+      { title: "foo", id: 6 },
+      { title: "foo", id: 7 },
+      { title: "foo", id: 8 },
+      { title: "foo", id: 9 }
+      ]
+    %}
+    {% for item in items %}
+      {% render '@vf-box--easy', {
```

Comment:
I meant `vf-box` not `vf-card` above.

Similarly here - need to check it's working fine with 11ty